### DISCUSSION TITLE
On hover of a field, a tooltip should be displayed with the field's description value if exists

### DISCUSSION THREAD

**Discussion by skyfox675** [twentyhq/twenty#5448] [Ideas]
## Scope & Context

As a user with a large data model, it would be helpful to quickly see a field's description value by hovering the field title.

**Comment by FelixMalfait** [comment]
Thanks for the suggestion! Moving to discussions as we won't do this short-term (most descriptions we wrote for standard objects are useless and so it would create a lot of noise if we implemented it now). But I agree this is something we should consider later!

**Reply by BOHEUS** [reply]
@FelixMalfait is it still relevant? I feel like it could be a nice first time contribution

**Comment by BOHEUS** [comment]
Added in #25408
